### DISCUSSION TITLE
Z-Stack Static Nav Bar Issue

### DISCUSSION THREAD

**Discussion by JessUVR** [google/model-viewer#3523] [General]
Hi everyone!

Does anybody have any experience with embedding the Model-Viewer component into a webpage with a fixed nav bar?

I am currently trying to create an AR Menu for a friend's restaurant and everything's was good until I added the Nav-Bar.
As I click on a dropdown menu I added the model seems to be higher on the Z-Stack hierarchy.

You need to use a phone in order to see the issue:
https://t0zyd9.csb.app/

Has anybody done something similar in the past?

**Comment by JessUVR** [comment]
![IMG_CE15FF726AC3-1](https://user-images.githubusercontent.com/32990384/175051771-d9968cb6-43d3-47e4-ac32-af3763e55e3d.jpeg)

This is the outcome I'm getting.

**Reply by elalish** [reply]
That is odd. Have you looked in dev tools yet? MV should respect CSS z-layer like any other div. Can you repro if you replace MV with an `<img>` tag?

**Reply by JessUVR** [reply]
Hi elalish!, Yes sir, I have looked in already, but couldn't find anything related to z-layer. As you can see in the two screenshots I'm sending to you in this message, when I add and `<img>` tag, the z-layer hierarchy is being respected.


![IMG_06729A18BBA7-1 2](https://user-images.githubusercontent.com/32990384/175208676-4f118f40-4b28-4fe0-a77d-8578f922d144.jpeg)
![IMG_06729A18BBA7-1](https://user-images.githubusercontent.com/32990384/175208699-085dd9eb-6718-47fc-88d9-08b0ca2c88e0.jpeg)
